CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Added Features
 
 - Telemetry collection enabled by default for self-managed installations. Opt-out is available on bundle generation, or at any time via the System Configuration UI.
+- A new environment variable `ROX_REGISTRY_MIRRORING_ENABLED` has been added that when set to `true` will enable processing registry mirrors during image enrichment.
```

Comment:
specifically as specified in OpenShift via the ImageContentSourcePolicy, right?

Reply:
Yes OpenShift - will also include `ImageDigestMirrorSet` and `ImageTagMirrorSet`